step-transifex-push.yml

```diff
@@ -0,0 +1,18 @@
+# Pushes string files to Transifex.
+
+parameters:
+  - name: string_file_pattern
+    type: string
+  - name: tx_token
+    type: string
+
+steps:
+  - bash: |
+      if [ `git log -m -1 --name-only --pretty="format:" | grep "${{ parameters.string_file_pattern }}" | wc -l` -gt 0 ]; then
```

Comment:
Interesting approach! I think the `git log` part is similar to the `git diff --name-only $target` + getting the [target](https://github.com/Doist/PipelinesTemplates/blob/master/step-ktlint-check-diff.yml#L15-L21). So basically getting the changed files, either from all commits for a PR or for one commit otherwise. Do you see advantages to either approach?

Reply:
Hmm, 🤔 The `git log` does not need to calculate the target so it's shorter. I don't see any other difference.